frontend/src/stores/ParticipantStore.js

```diff
@@ -161,10 +165,17 @@ export class ParticipantStore {
   @action setSidebarView = sidebarView => {
     this.sidebarView = sidebarView
   }
-  @action setSnackbarState = (message, options = { open: true }) => {
-    // we setup otions like this to maybe add more later
-    this.snackbarState.message = message
-    this.snackbarState.open = options.open
+  @action setSnackbarState = ({
+    message = "",
+    severity = SNACKBAR_SEVERITY.INFO,
+    open = true,
+  }) => {
+    // prevent visual change to severity or message when closing the snackbar
```

Comment:
was the message /severity changing before it disappeared?

Reply:
Yes. There is an animation when it disappears, but the severity and message change happens instantly.